src/plugins/opensearch_dashboards_overview/common/index.ts

```diff
@@ -30,5 +30,5 @@
 
 export const PLUGIN_ID = 'opensearchDashboardsOverview';
 export const PLUGIN_NAME = 'Overview';
-export const PLUGIN_PATH = `/app/opensearch_dashboards_overview`;
+export const PLUGIN_PATH = '';
```

Comment:
This change has to be reverted as it's causing issues with the `discover` app.

![image](https://user-images.githubusercontent.com/15186973/233598452-5692a7a4-55f1-4786-83d3-2236b067c369.png)